MapboxNavigation/Dictionary.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
+    /**
+     Returns a proportional `Dictionary` of `lineWidthAtZoomLevels` at a specific factor.
```

Comment:
As written, this method could just as well operate on circle size stops, not just line width stops:

> Returns a copy of the stop dictionary with each value multiplied by the given factor.